amethyst_ui/src/text_editing.rs

```diff
@@ -266,6 +287,40 @@ impl<'a> System<'a> for TextEditingInputSystem {
                                 }
                             }
                         }
+                        VirtualKeyCode::Return | VirtualKeyCode::NumpadEnter => {
+                            match focused_text.line_mode {
+                                LineMode::Single => {
+                                    edit_events
+                                        .single_write(UiEvent::new(UiEventType::Commit, entity));
+                                }
+                                LineMode::Wrap => {
+                                    if modifiers.shift
```

Comment:
Shift+Enter at max length = commit? Shouldn't this be ignored?

Reply:
I would generally expect shift + enter to make a newline, and enter alone to commit (If I understand correctly what commit means).  Maybe the behavior here should be controllable through options/flags?